docs/running-on-yarn.md

```diff
@@ -134,6 +134,13 @@ Most of the configs are the same for Spark on YARN as for other deployment modes
   </td>
 </tr>
 <tr>
+  <td><code>spark.yarn.am.port</code></td>
+  <td>(random)</td>
+  <td>
+    Port for the YARN Application Master to listen on.
```

Comment:
I am not 100% sure but I think it does communicate with driver also, in runAMEndpoint method it is creating RpcEndpoint to communicate with driver. Can someone please confirm, I will update description accordingly.